loki/transformations/inline/tests/test_inline_transformation.py

```diff
@@ -338,18 +339,24 @@ def test_inline_transformation_adjust_imports(frontend, tmp_path):
   implicit none
   contains
 
-subroutine test_inline_inner(a, b)
+subroutine test_inline_inner(a, b, c, d)
   use BNDS_module, only: n, m
   use another_module, only: x
 
   real(kind=8), intent(inout) :: a(n), b(n)
+  real(kind=8), intent(out) :: c(4), d(4)
   real(kind=8) :: tmp(m)
   integer :: i
 
   tmp(1:m) = x
   do i=1, n
     a(i) = b(i) + sum(tmp)
   end do
+  do i=1,4
+    c(i) = 0.
```

Comment:
Could we add an assignment `d(i) = 1.` here too, to check that this index is shifted too?